frontend/pages/schedule/ManageSchedulePage/components/ScheduleListWrapper/ScheduleListWrapper.tsx

```diff
@@ -35,6 +41,7 @@ interface IScheduleListWrapperProps {
   inheritedQueries?: boolean;
   isTeamMaintainerOrTeamAdmin: boolean;
   isOnGlobalTeam: boolean;
+  selectedTeamData: ITeam | undefined;
```

Comment:
Same here, can we remove `teamId`?

Reply:
Could this be `selectedTeam` instead of `selectedTeamData`? Do we have a convention of adding `Data` to the end of any other props?

Reply:
I looked into changing this and 80% of the way through I stumbled on why I did it... line 245 - 249 on the ManageSchedulesPage.tsx uses
`  const selectedTeam = isNaN(teamId) ? "global" : teamId;

  const selectedTeamData = allTeamsList.find(
    (team) => selectedTeam === team.id
  );`
  
  There's already a `selectedTeam` for the team dropdown

Reply:
Makes sense!